client/src/components/InputsModal/InputOAuthCredentials.tsx

```diff
@@ -63,46 +52,70 @@ const InputOAuthCredentials: FC<InputOAuthCredentialsProps> = ({
     </Fragment>
   );
 
-  const oAuthField = (field: InputOAuthField) => (
-    <ListItem disabled={requirement.locked} key={field.name}>
-      <TextField
-        disabled={requirement.locked}
-        id={`requirement${index}_${field.name}`}
-        label={field.label || field.name}
-        helperText={requirement.description}
-        value={oAuthCredentials[field.name]}
-        className={styles.inputField}
-        variant="standard"
-        fullWidth
-        onChange={(event) => {
-          const value = event.target.value;
-          inputsMap.set(requirement.name, value);
-          oAuthCredentials[field.name] = value;
-          inputsMap.set(requirement.name, JSON.stringify(oAuthCredentials));
-          setInputsMap(new Map(inputsMap));
-        }}
-        InputLabelProps={{ shrink: true }}
-      />
-    </ListItem>
-  );
+  const oAuthFields: InputOAuthField[] = [
+    { name: 'access_token', label: 'Bearer Token', required: true },
```

Comment:
I think we should label this `Access Token`.

Reply:
Done!